training_scripts/gsgnn_np/arxiv_nc.yaml

```diff
@@ -16,8 +16,8 @@ gsf:
   input:
     restore_model_path: null
   output:
-    save_model_path: /data/ogbn-arxiv-1p/model/
-    save_embed_path: /data/ogbn-arxiv-1p/embed/
+    save_model_path: null
+    save_embed_path: null
```

Comment:
Are these changes related to this PR? Or just a drive-by fix?

Reply:
It is a drive-by fix.